src/main/java/com/bina/varsim/fastqLiftover/readers/ARTFastqAlnReader.java

```diff
@@ -23,11 +26,23 @@ public ARTFastqAlnReader(final BufferedReader alnR, final BufferedReader fastqBr
     public SimulatedRead getNextRead() throws IOException {
         SimulatedRead read;
         final String nameLine = fastqBr.readLine();
-        if (nameLine == null) return null;
+        if (nameLine == null) {
+            return null;
+        }
+        if (nameLine.trim().length() < 1) {
+            log.warning("got empty string at FASTQ line " + fastqBr.getLineNumber());
```

Comment:
Should also print the out the actual line along with the line number. Shouldn't this be an error instead of just a warning?